### PR TITLE
Add POOL token on Optimism to token list

### PR DESCRIPTION
**Pull request by chuckbergeron**
Add POOL token on Optimism to token list

### CODE REVIEW

File: optimism.tokenlist.json [hunk 1]
```diff
@@ -1728,7 +1728,29 @@
...
+      "chainId": 10,
+      "address": "0x395ae52bb17aef68c2888d941736a71dc6d4e125",
+      "name": "PoolTogether",
+      "symbol": "POOL",
+      "decimals": 18,
+      "logoURI": "https://ethereum-optimism.github.io/logos/pool.svg",
+      "extensions": {
+        "optimismBridgeAddress": "0x4200000000000000000000000000000000000010"
+      }
+    },
```
Comment (roninjin10) [inline optimism.tokenlist.json:1753]: syntax error is failing the tests.  Please remove this comma

**Review by roninjin10 — CHANGES_REQUESTED** [review]
Syntax error failing tests

**Comment by Nicca42** [comment]
We also only need one image, recommend leaving the SVG and deleting the PNG.

**Comment by chuckbergeron** [comment]
@Nicca42 @roninjin10 great, thanks. Those have been fixed!

**Review by roninjin10 — APPROVED** [review]

**Comment by chuckbergeron** [comment]
@roninjin10 I believe this is good to go now. We noticed there was an issue with the `validate` task due to the Optimism bridge address I used being lowercase instead of checksummed. I don't see it running again on GitHub actions, but it's running fine for me locally. Thanks again.

**Comment by roninjin10** [comment]
@chuckbergeron the lowercase thing has been fixed.   Check is failing for odd reason.  Can you merge your changes with latest master and try again?

**Comment by chuckbergeron** [comment]
@roninjin10 Ok done. Could we give the GitHub action another go? This branch is acting wonky for me. If it doesn't work this time I'll close this PR and create a new branch & PR. 👍

**Comment by roninjin10** [comment]
tyty @chuckbergeron

**Pull request merged**